_pages/about.md

```diff
@@ -12,32 +12,25 @@ toc_sticky: true
 toc_icon: "align-justify"
 ---
 
-# Who we are
-
-We are Bjoern and Kirthihan, two enthusiastic computer engineers with a passion
-for embedded systems, digital design and efficient software programming. Our
-journey together began when we studied electrical engineering at RWTH Aachen
-University in Germany. At the Institute for Communication Technologies and
-Embedded Systems, we worked as student assistants, where we contributed to
-different projects in the fields of hardware accelerators for machine learning,
-FPGA design, processor prototyping and embedded software. Nowadays, Bjoern works
-as a firmware developer in the audio industry. Kirthihan works as a digital
-hardware developer in the cybersecurity industry.
-
-# What you can expect
+# Bits&Volts - A blog for everyone
 
 There are already tons of good resources available. So you are probably
-wondering why we are starting another blog. On the one hand, we love to work
-together on electronic projects. On the other hand, we like to share our
-knowledge with other people. That's why we created Bits&Volts.
+wondering why we are starting another blog. Back when we were fresh students, we
+always felt that the barrier to enter the field of electronics is way higher
+than it should've been. Even at university, we learned deep and complex theory,
+while simple and practical aspects were rarely explained. So many times we did
+things wrong or inefficiently while a voice in our head asked: "Why didn't
+anyone tell us?". It can be hard to get going with electronics since you do not
+know the right place to start. How can you distinguish between a good and a bad
+resource? Often we come across blogs or other kinds of resources that either do
```

Comment:
Hm, I'm still not a fan of "blog" here. Resource would be enough, IMO.

Unless you know of a good example real world for a "bad blog".

But if you think that it is fine, leave it.

Reply:
I tried to avoid resource since it is the last word of the previous sentence. Any other idea?

Reply:
I think the repetition is acutally not that bad here. It is like if the second sentence picks up "resource" from the previous one and then elaborates on it.
```suggestion
resource? Often we come across resources that either do ...
```